core/src/autogluon/core/features/infer_types.py

```diff
@@ -136,3 +136,26 @@ def check_if_nlp_feature(X: Series) -> bool:
         return False
 
     return True
+
+
+def get_bool_true_val(series: pd.Series, replace_val=None):
+    """
+    From a pandas series, get the replace_val to convert to boolean when calling:
+    series_bool = series == replace_val
+
+    series must have exactly 2 unique values
+    """
+    if replace_val is None:
+        try:
+            uniques = series.unique()
+            uniques.sort()
+        except:
+            uniques = series.unique()
```

Comment:
Effectively this:
```python
        uniques = series.unique()
        try:
            uniques.sort()
        except:
            pass # log something?
```

Reply:
Yes, I did it how it is currently because .sort() is an inplace operation and I was unsure if calling it (and then erroring) would lead to an altered uniques object.

Reply:
added comments